internal/v1/handler.go

```diff
@@ -300,7 +308,7 @@ func parseComposeStatusError(composeErr *composer.ComposeStatusError) *ComposeSt
 }
 
 func (h *Handlers) GetComposeMetadata(ctx echo.Context, composeId uuid.UUID) error {
-	err := h.canUserAccessComposeId(ctx, composeId)
+	_, err := h.canUserAccessComposeId(ctx, composeId)
```

Comment:
Granted this is sub-ideal, I offer refactoring into special SQL later as a separate PR. I think the fasted way would be something like `SELECT ID FROM COMPOSES WHERE XXX LIMIT 1` I guess. Maybe `COUNT`.

Reply:
Sounds good!